.maintain/setup-darwinia.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+#
+#
+
+set -xe
+
+BIN_PATH=$(dirname $(readlink -f $0))
+WORK_PATH=${BIN_PATH}/../
+
+
+echo -e '\e[1;32m📥 Installing Cross Compile Toolchain(s)\e[0m'
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | \
+  sh -s -- -y --profile minimal --default-toolchain ${RUST_TOOLCHAIN}
+source ~/.cargo/env
+
+
+if [ -n "${OVERALL_TEST}" ]; then
+  cargo test
+  exit 0
+fi
+
+
+echo -e '\e[1;32m🔧 Building Docker Image(s)\e[0m'
+docker build -f .maintain/docker/Dockerfile.x86_64-linux-gnu -t x86_64-linux-gnu .
+docker build -f .maintain/docker/Dockerfile.aarch64-linux-gnu -t aarch64-linux-gnu .
+
+
+cargo install cross \
+  --git https://github.com/AurevoirXavier/cross \
+  --branch support-multi-sub-targets
+
+rustup target add \
+  x86_64-unknown-linux-gnu \
+  aarch64-unknown-linux-gnu \
+  wasm32-unknown-unknown
+
+
+cross build \
+  --release \
+  --target x86_64-unknown-linux-gnu \
+  --sub-targets wasm32-unknown-unknown
+
+#RUSTFLAGS='-C link-args=-latomic' SKIP_WASM_BUILD=1 cross build \
+#  --no-default-features \
+#  --locked \
+#  --release \
+#  --target aarch64-unknown-linux-gnu
+
+# deploy folder is required,
+# the x86_64 file will move to this folder,
+# the build-image step will use this file to build a docker image
+mkdir -p ${WORK_PATH}/deploy/bin
+
+cd ${WORK_PATH}/deploy/bin/
+
+cp ${WORK_PATH}/target/x86_64-unknown-linux-gnu/release/darwinia ${WORK_PATH}/deploy/bin/
+tar cjSf darwinia-x86_64-linux-gnu.tar.bz2 darwinia
+mv darwinia ${WORK_PATH}/deploy/
+
+#cp ${WORK_PATH}/target/aarch64-unknown-linux-gnu/release/darwinia ${WORK_PATH}/deploy/bin/
```

Comment:
There I have a question for @AurevoirXavier , in .maintain/utility/release script, after build use aarch64 why copy file use x86_64 folder? 
https://github.com/darwinia-network/darwinia/blob/d1247dc8697bbba397d96b29e40e95106b29cd18/.maintain/utility/release#L37-L39
It's a bug? if that, I can release this comment to support aarch64 build in this script

Reply:
Seems we made a mistake here.

Reply:
And could you remove the old releases' aarch64 binary.

Reply:
ok, I'll do last test in my repository enable aarch64, and update this script, when build finish, I'll post release url to here.

Reply:
> 
> 
> And could you remove the old releases' aarch64 binary? If you don't have the permission let me do that.

You can do this, I don't know how to remove that